### ISSUE TITLE
 Create a W&B account？

### ISSUE THREAD

**Issue by watertianyi** [jiyegege/animeGanv2_pytorch#1]
AttributeError: 'VGGCaffePreTrained' object has no attribute 'features'
wandb: Waiting for W&B process to finish... (failed 1).
wandb: You can sync this run to the cloud by running:
wandb: wandb sync ./wandb/offline-run-20230318_194510-3tktpfql
wandb: Find logs at: ./wandb/offline-run-20230318_194510-3tktpfql/logs

**Comment by jiyegege** [comment]
Yes, you need to have a W&B account and log in to your account using `wand login` before running the program

**Comment by watertianyi** [comment]
@jiyegege why do you do this？

**Comment by jiyegege** [comment]
@seawater668 If you don't need W&B, you can also change the code to cancel this way, W&B is just to facilitate me to record training loss。you also can use `wandb offline` disable W&B sync

**Comment by watertianyi** [comment]
@jiyegege AttributeError: 'VGGCaffePreTrained' object has no attribute 'features'

**Comment by jiyegege** [comment]
@seawater668 what’s your Python version，my version is 3.8.16

**Comment by watertianyi** [comment]
@jiyegege Python 3.8.16

**Comment by jiyegege** [comment]
@seawater668 I have run the code on both my macos and liunx machines, and there is no error in your feedback. But according to your feedback, the code error may appear in the `backtone.py` file, have you changed this file
<img width="866" alt="image" src="https://user-images.githubusercontent.com/26988300/226786789-c8424d58-ad89-4eb0-b7fb-7c86457b58cf.png">

**Comment by watertianyi** [comment]
@jiyegege 
This file has not been changed, does vgg19.npy need to be downloaded in advance

**Comment by watertianyi** [comment]
@jiyegege 
 File "train.py", line 123, in main
    model.to_onnx('animeGan.onnx', input_sample=torch.randn(1, 3, 256, 256))

RuntimeError: ONNX export failed: Couldn't export operator aten::upsample_bilinear2d

**Comment by watertianyi** [comment]
After training for 40 epochs, the test result does not change the real face. What I am training is the cartoonization of the face.

**Comment by jiyegege** [comment]
@seawater668 If you're just stylizing faces, you'll probably need to make your own data sets and extract face parts for stylizing. There's animeGan's pytorch open source project on github for faces

**Comment by watertianyi** [comment]
The face part has been extracted
